### PR TITLE
feat(plugin-essentials): config unset

### PR DESCRIPTION
**Pull request by chengcyber**
**What's the problem this PR addresses?**

There is currently no way to unset a config from the CLI

**How did you fix it?**

Add a `config unset` command

**Checklist**

- [x] I have read the [Contributing Guide](https://yarnpkg.com/advanced/contributing).

- [x] I have set the packages that need to be released for my changes to be effective.

- [x] I will check that all automated PR checks pass before the PR gets reviewed.

### CODE REVIEW

File: .yarn/versions/630f0472.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/plugin-essentials": patch
```
Comment (merceyz) [inline .yarn/versions/630f0472.yml:3]:
```suggestion
  "@yarnpkg/cli": minor
  "@yarnpkg/plugin-essentials": minor
```

**Comment by arcanis** [comment]
All good 🚀

**Comment by merceyz** [comment]
Good stuff :rocket:

File: packages/plugin-essentials/sources/commands/config/unset.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  @Command.String()
+  name!: string;
+
+  @Command.Boolean(`-H,--home`, {description: `Update the home configuration instead of the project configuration`})
+  home: boolean = false;
+
+  static usage: Usage = Command.Usage({
+    description: `unset a configuration settings`,
+    details: `
+      This command will set a configuration setting.
```
Comment (merceyz) [inline packages/plugin-essentials/sources/commands/config/unset.ts:18]: Needs to be updated as it doesn't set a config

File: packages/plugin-essentials/sources/commands/config/unset.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      `Unset a simple configuration setting`,
+      `yarn config set initScope`,
+    ], [
+      `Unset a complex configuration setting`,
+      `yarn config set packageExtensions`,
+    ], [
+      `Unset a nested configuration setting`,
+      `yarn config set npmScopes.company.npmRegistryServer`,
+    ]],
+  });
```
Comment (merceyz) [inline packages/plugin-essentials/sources/commands/config/unset.ts:30]: This needs to be updated, amazing we didn't notice
(thanks @andreialecu )

**Pull request merged**